Makefile

```diff
@@ -3,33 +3,18 @@
 # SPDX-License-Identifier: MPL-2.0
 ts := $(shell date +"%Y%m%d%H%M")
 
```

Comment:
this absolutely needs PHONY if you're going to use makefiles like this

Reply:
fixed phony in #18